setup.cfg

```diff
@@ -28,18 +28,23 @@ exclude =
 # D400 First line should end with a period
 # D401 First line should be in imperative mood
 # D403 First word of the first line should be properly capitalized
+# DAR101 Missing parameter(s) in Docstring: - min_hits
```

Comment:
Yeah, it is a good check but if there are multiple, it turns messy quite quickly